kornia/color/raw.py

```diff
@@ -211,6 +211,64 @@ def rgb_to_raw(image: torch.Tensor, cfa: CFA) -> torch.Tensor:
     return output
 
 
+def raw_to_rgb_2x2_downscaled(image: torch.Tensor, cfa: CFA) -> torch.Tensor:
```

Comment:
```suggestion
def raw_to_rgb_2x2_downscaled(image: Tensor, cfa: CFA) -> Tensor:
```
`from kornia.core import Tensor`

Reply:
Done